graviti/client/branch.py

```diff
@@ -8,7 +8,7 @@
 from typing import Any, Dict, Optional
 from urllib.parse import urljoin
 
-from graviti.utility import URL_PATH_PREFIX, open_api_do
+from graviti.client.requests import URL_PATH_PREFIX, open_api_do
```

Comment:
we have more than one commit scope in this commit: client + openapi + utility,
in this case, we can remove the scope in commit message

Reply:
we have more than one requests.py file in our repo, it will be better to point out the requests is `utility/requests.py`
or `utility.requests`